pkg/minikube/cluster/cluster.go

```diff
@@ -618,3 +618,17 @@ func GetServiceURLs(api libmachine.API, namespace string, t *template.Template)
 
 	return serviceURLs, nil
 }
+
+//Create the host directories in the virtual machine if not present
+func EnsureHostDirectories(api libmachine.API, dirs []string) (error) {
```

Comment:
I think we should move this somewhere else. _cluster.go_ is big enough as it stands. We could have a vmutils package of sorts, since this method for example is not specific per se to the cluster functionality. It just ensures that a set of directories exit.

Alternatively, we can create _oc_cluster.go_ somewhere under _pkg/minishift/oc/cluster_. This could be where we put other _oc cluster_ stuff as well (see issue #409).